scripts/r.in.wms/wms_gdal_drv.py

```diff
@@ -60,7 +60,6 @@ def _createXML(self):
 
         gdal_wms = ET.Element("GDAL_WMS")
         service = ET.SubElement(gdal_wms, "Service")
-        name = ET.Element("name")
```

Comment:
That seems weird to me. What was it supposed to do? The pattern of the line below, that sets an attribute "name" on service, but an element name is made is weird

Reply:
I think this line is perhaps a mistake in the original code, so I am fine with removing it.

Reply:
With the docs linked, it really looks like with that line removed it does the correct thing.

Reply:
The line `service.set("name", "WMS")` creates `<Service name="WMS">`. I think it is OK to remove the line.